bookkeeper-common/src/test/java/org/apache/bookkeeper/common/net/ServiceURITest.java

```diff
@@ -206,6 +206,30 @@ public void testServiceInfoPlus() {
             "/path/to/namespace");
     }
 
+    @Test
```

Comment:
I think it's better to test the IPV6 with all the current test cases.

Reply:
Done